applications/conversations/controllers/api/MessagesApiController.php

```diff
@@ -118,7 +118,7 @@ private function fullSchema() {
                 'messageID:i' => 'The ID of the message.',
                 'conversationID:i' => 'The ID of the conversation.',
                 'body:s' => [
-                    'maxLength' => $this->config->get('MaxLength', 2000),
+                    'maxLength' => $this->config->get('Conversations.Message.MaxLength', 2000),
```

Comment:
I think you're only going to want the length restrictions to be applied to the write schemas. If you're applying them to all the schemas, then you're running the risk of having existing conversation messages break validation in API responses after decreasing the maximum length.